docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -945,12 +945,22 @@ peerCertificateValidated()
 
 .. code-block:: lua
 
-  if downstreamSslConnection:peerCertificateVaidated() then
-    print("peer certificate is valiedated")
+  if downstreamSslConnection:peerCertificateValidated() then
+    print("peer certificate is validated")
   end
 
 Returns bool whether the peer certificate was validated.
 
+.. warning::
+
+   Client certificate validation is not currently performed upon TLS session resumption. For a
```

Comment:
Is there a bug to track the overall issue? In theory, you could save the validation status into the session ticket, no?
Also, what about `presented`? Is that also affected?

Reply:
Yes, there is #21235. Should the docs link to this bug?

If I'm reading the BoringSSL source correctly, there is in fact a field in the session ticket structure for storing validation status (based on [this comment](https://boringssl.googlesource.com/boringssl/+/d94be4172325d0ac239ef3046d2c609aebeaf00a/ssl/ssl_asn1.cc#113)), but I don't see that the current validation callback has any way to store a validation error in this field while still accepting a connection. I think support for this may require changes within BoringSSL, but please let me know if I'm missing something.

I don't believe `presented` is affected. BoringSSL does store the entire certificate chain in the session ticket, so it is still available after session resumption. (I've observed that the Lua method `urlEncodedPemEncodedPeerCertificateChain()` still returns the same results after a session resumption.)